src/serlo-editor-repo/plugin-text/components/suggestions.tsx

```diff
@@ -1,89 +1,80 @@
 import React from 'react'
 
 import { styled } from '../../ui'
-import type { TextEditorPluginConfig } from '../types'
+import type { TextEditorPluginConfig, Theme } from '../types'
+import type { RegistryPlugin } from '@/serlo-editor-repo/plugin-rows'
 
 interface SuggestionsProps {
   config: TextEditorPluginConfig
-  options: { name: string; title?: string }[]
-  currentValue: string
+  options: RegistryPlugin[]
+  suggestionsRef: React.MutableRefObject<HTMLDivElement | null>
   selected: number
   onMouseDown: (option: string) => void
 }
 
-function escapeRegExp(string: string) {
-  return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')
-}
+const SuggestionsWrapper = styled.div({
+  maxHeight: '200px',
+  maxWidth: '300px',
+})
 
-const Suggestion = styled.div<{ isActive: boolean }>(({ isActive, theme }) => ({
-  height: '32px',
-  padding: '4px 8px',
+const Suggestion = styled.div(({ theme }: { theme: Theme }) => ({
+  padding: '5px 12px',
   cursor: 'pointer',
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-  backgroundColor: isActive
-    ? // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-      theme.suggestions.background.highlight
-    : // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-      theme.suggestions.background.default,
-  borderRadius: '4px',
-  '&:hover': {
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
-    background: theme.suggestions.background.highlight,
+  display: 'flex',
+  alignItems: 'start',
+  backgroundColor: theme.suggestions.background.default,
+  '&:hover, &[data-active="true"]': {
+    backgroundColor: theme.suggestions.background.highlight,
+    [SuggestionIconWrapper]: {
+      border: '0.1px solid #404040',
+    },
   },
 }))
 
-const Container = styled.div({
-  padding: '10px',
+const SuggestionIconWrapper = styled.div({
+  border: '0.1px solid transparent',
```

Comment:
see above

Reply:
Yeah, I just copy/pasted from Figma and later changed the color, didn't really pay attention to the width :D